src/args.rs

```diff
@@ -1,4 +1,7 @@
 use clap::Parser;
 
 #[derive(Parser)]
-pub struct Args {}
+pub struct Args {
+    #[arg(long)]
+    pub rule: Option<String>,
```

Comment:
I exposed this so that the testing could currently set up an environment of just running the one rule under test for one file